shared/src/validation/eam-worker/worker-create/worker-create.validation-schema.ts

```diff
@@ -18,6 +18,17 @@ const EamWorkerCreate = Joi.object({
       'string.min': EAMWorkerValidationMessage.NAME_MIN_LENGTH,
       'string.pattern.base': EAMWorkerValidationMessage.NAME_REGEX,
     }),
+  [getNameOf<EAMWorkerCreateRequestDto>('password')]: Joi.string()
```

Comment:
we use this validation for the frontend. there is no `password` field. you need to create a separate one